Framework/MPI/inc/MantidMPI/Helpers.h

```diff
@@ -0,0 +1,57 @@
+#ifndef MANTID_MPI_HELPERS_H_
```

Comment:
I'm not a big fan of helpers either functions, headers or classes. Could a more descriptive name be used, or could this be broken up into separate headers if needed.